examples/animation.html

```diff
@@ -2,7 +2,7 @@
 <head>
   <title>Share Button Collision Test</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link href="dist/styles.min.css" rel="stylesheet">
+  <link href="dist/ShareButtonStyles.min.css" rel="stylesheet">
```

Comment:
should probably be `share-button.min.css`, because urls/files usually use hyphenation and because `.css` already tells you it's styles